index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      <div class="title">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="" id="header-img">
+      </div>
+      <nav id="nav-bar"><ul>
+        <li class="nav-link"><a href="#services">Features</a></li>
+        <li class="nav-link"><a href="#">How It works</a></li>
+        <li class="nav-link"><a href="#">Pricing</a></li>
+      </ul></nav>
+    </header>
+    <main>
+      <section id="header-content">
+        <h2>Handcrafted, home-made masterpieces</h2>
```

Comment:
I think it would be better to use an h1 tag for the text in the main section 😉

Reply:
Good job @over-geek , your comment is next to the adequate line of code and your message is clear.
[ OPTIONAL ] it's good to add links or references to a document explaining why they should do so this way the student knows that the comment is not a personal point of view .